cmd/runcommand.go

```diff
@@ -0,0 +1,34 @@
+package cmd
```

Comment:
I'm not sure about this project formatting rules but you could use gofmt on this file.